service/test/AuthorizationProcessorImplTest.java

```diff
@@ -1,176 +0,0 @@
-import com.auth0.jwt.JWT;
```

Comment:
you sure we want to delete this whole spec?

Reply:
Yeah, we got rid of it in the other repos, and it's in the wrong location to even run. Do you want me to add it  back in and move it to where it should be?

Reply:
If you think it's easier then yeah. Up to you.

Reply:
I'll add it back in if it still runs. Last I remember there were a lot of changes, but we'll see.

Reply:
Yeah, it looks like things are completely different now.